.eslintrc.js

```diff
@@ -0,0 +1,241 @@
+'use strict';
+
+const Module = require('module');
+const path = require('path');
+
+const NodePlugin = require('./tools/node_modules/eslint-plugin-node-core');
+NodePlugin.RULES_DIR = path.resolve(__dirname, 'tools', 'eslint-rules');
+
+const ModuleFindPath = Module._findPath;
+const hacks = ['eslint-plugin-node-core', 'eslint-plugin-markdown', 'babel-eslint'];
+Module._findPath = (request, paths, isMain) => {
+  const r = ModuleFindPath(request, paths, isMain);
+  if (!r && hacks.includes(request)) {
+    try {
+      return require.resolve(`./tools/node_modules/${request}`);
+    } catch (err) {
+      return require.resolve(
+        `./tools/node_modules/eslint/node_modules/${request}`);
+    }
+  }
+  return r;
+};
+
+module.exports = {
```

Comment:
I think it makes sense to put these into a separate file (either a JSON or a .js), and re-export that from here. That way we can separate the hack from the actual rules.

Reply:
i'd rather not just for simplicity's sake